task_2/andrii.lukianchykov/task_2/task_2/task_2/src/main/java/com/bookshop/bookdatabase/BookDataBase.java

```diff
@@ -0,0 +1,85 @@
+package com.bookshop.bookdatabase;
+
+import com.bookshop.book.Book;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.System.out;
+
+public class BookDataBase {
+
+    private Book book;
```

Comment:
You don't need a book object itself in that class.

Reply:
If I don`t need this object, how can i manage with my method getBook()?

Reply:
Or I need just use the Book book = new Book() , or create object exactly in class Book?